api/src/main/java/marquez/service/models/JobEvent.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2018-2023 contributors to the Marquez project
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package marquez.service.models;
+
+import java.net.URI;
+import java.time.ZonedDateTime;
+import java.util.List;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Setter
+@Getter
+@Valid
+@ToString
+public class JobEvent extends BaseEvent {
```

Comment:
Do we have to define our own OL events, or can we use the [`io.openlineage.server`](https://openlineage.io/apidocs/javadoc/io/openlineage/server/package-summary.html) models?

Reply:
I would love to see whole Marquez `LineageEvent` being rewritten `io.openlineage.server` models at some point. I thought it does not make much sense to do it partially.

Reply:
Agreed. It's been on my radar to switch over for a while and we'll get to it at some point. Follow up PR it is 👍